### PR TITLE
Replace WinUI3GetStarted 2020 project by result of latest project template

### PR DESCRIPTION
**Pull request by mikehoffms**
To create this PR, I did the following:
1. Tested & updated the article in a Docs PR.
2. Replaced the present repo's project by the project that resulted from the steps in that updated article, updating project from VS2019 (2020) to VS2022 (2024).  In Files Changed tree:
   * New dir is added: `WinUI3GetStarted`
   * Old dir is deleted: `WinUI_Sample`

Related PR in Docs repo: [PR 3297](https://github.com/MicrosoftDocs/edge-developer/pull/3297) (Add "Update Windows App SDK" in "Get started WinUI 3 (Win App SDK)")

### CODE REVIEW

File: GettingStartedGuides/WinUI3_GettingStarted/README.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ languages:
 page_type: sample
 products: 
   - microsoft-edge
-urlFragment: WinUI3_GettingStarted
+urlFragment: WinUI3GetStarted
```
Comment (mikehoffms) [inline GettingStartedGuides/WinUI3_GettingStarted/README.md:8]: revert this change, if this represents parent/containing dir

Comment (mikehoffms) [inline GettingStartedGuides/WinUI3_GettingStarted/README.md:8]:
```suggestion
urlFragment: WinUI3_GettingStarted
```

**Review by peiche-jessica — APPROVED** [review]
